plugins/VstEffect/VstEffect.cpp

```diff
@@ -70,7 +70,7 @@ VstEffect::VstEffect( Model * _parent,
 		openPlugin( m_key.attributes["file"] );
 	}
 	setDisplayName( m_key.attributes["file"].section( ".dll", 0, 0 ).isEmpty()
-		? m_key.name : m_key.attributes["file"].section( ".dll", 0, 0 ) );
+		? m_key.name : m_key.attributes["file"].section( ".dll", 0, 0 ).section("\\", -1).section("/", -1));
```

Comment:
It can also end with. `.so` with native Linux vst's.

Reply:
What changes need to be done to add .so to the list?

Reply:
I don't know. There are qt functions to handle filename/paths, like https://doc.qt.io/qt-6/qfileinfo.html#baseName

Reply:
Oh, we're not stripping a part of the actual file name but an attribute of a file. Then I don't know.

Reply:
Now since you said it, looks like a hacky way more than an actual fix. Also looks like something can break somewhere.

Reply:
If you 'grep' the lmms code for effGetEffectName, you may find some clues. Unfortunately I just borked my vst chain so I'm a bit blind sided there right now.